ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -853,6 +853,33 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: aws-ipi-ovn-hypershift-guest-p1-critical-f7
```

Comment:
no longer need `pN` in the test name.

Reply:
@liangxia Thanks for your reminder, could you help to review it again?